### PR TITLE
Enable update on user switchboards

### PR DESCRIPTION
**Pull request by xxbillouxx**

### CODE REVIEW

**Comment by xxbillouxx** [comment]
recheck

File: wazo_confd/plugins/switchboard_member/middleware.py [hunk 1]
```diff
@@ -26,3 +30,36 @@ def associate(self, body, switchboard_uuid, tenant_uuids):
             raise errors.param_not_found('users', 'User', **e.metadata)
 
         self._service.associate_all_member_users(switchboard, users)
+
+    def dissociate(self, user_id, switchboard_uuid, tenant_uuids):
+        """
+        Add a new user to the members of a switchboard.
```
Comment (pc-m) [inline wazo_confd/plugins/switchboard_member/middleware.py:36]: Are you sure this comment is right?

Comment (pc-m) [reply wazo_confd/plugins/switchboard_member/middleware.py:36]: or maybe the name of the function should not be dissociate?

File: wazo_confd/plugins/user/resource.py [hunk 2]
```diff
@@ -72,7 +72,16 @@ def get(self, id):
 
     @required_acl('confd.users.{id}.update')
     def put(self, id):
-        return super().put(id)
+        tenant_uuids = self._build_tenant_list({'recurse': True})
+        self._middleware.update(
+            id,
+            request.get_json(),
+            tenant_uuids,
+            recursive=request.args.get(
```
Comment (pc-m) [inline wazo_confd/plugins/user/resource.py:80]: It's not the first time I see this line. It should be extracted to a function

**Review by pc-m — CHANGES_REQUESTED** [review]

**Review by pc-m — APPROVED** [review]

**Pull request merged**